backend/overstroomik_service/config.py

```diff
@@ -8,7 +8,21 @@ class Settings(BaseSettings):
     # e.g: '["http://localhost", "http://localhost:4200", "http://localhost:3000", \
     # "http://localhost:8080", "http://local.dockertoolbox.tiangolo.com"]'
     BACKEND_CORS_ORIGINS: List[AnyHttpUrl] = []
-    PORT: int = 8000        
+    PORT: int = 8000
+    GEOSERVER_URL: str = "http://geoserver:8080/geoserver"
+    GEOSERVER_LAYER: str = "overstroomik:Overstroomik_data"
+
+    LAYER_EVACUATION_PERCENTAGE: str  = "evacuatie_evacuatiefractie_verwachtingswaarde"
```

Comment:
Very minor nitpick, all these lines have two spaces after the type, 1 would be normal.

Reply:
Deleted one of the two spaces.